### PR TITLE
Prasad_C_Coding_exam_4_29mins

### PR DESCRIPTION
**Pull request by PrasadCOct2020**
Hi Harsh,

Please review my PR 
image

image

Regards 
Prasad Chitale

### CODE REVIEW

File: src/prasad/Findcommonelefrarray.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package prasad;
+
+public class Findcommonelefrarray {
+	void scanArray(int[]array1 , int[]array2) {
+		if(array1.length!=array2.length)
+			System.out.println("Given two arrays are not of same length, please retry");
+		else
+			System.out.println("Common Elements:- ");
+		for(int index=0;index<array1.length;index++) {
+			if(array1.length != array2.length)
```
Comment (harshvegada) [inline src/prasad/Findcommonelefrarray.java:10]: this condition you already given, then why again here.

Comment (harshvegada) [reply src/prasad/Findcommonelefrarray.java:10]: Hi Prasad,
you need to compare element in two loop, fortunately both array have same common value on same index, but think on diff index,
Pls update accordingly. I know this is not mentioned but this is like counter question on same program.

File: src/prasad/Findiftwostrareindentical.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+public class Findiftwostrareindentical {
+	
+	boolean scanStr(String input1 , String input2) {
+		boolean flag = false;
+		int count=0;
+		for(int index=0 ; index<input1.length(); index++) {
+			if(input1.length()!=input2.length() || input1.charAt(index)!=input2.charAt(index))
+				break;
+			
+			else if(input1.charAt(index)== input2.charAt(index)) {
```
Comment (harshvegada) [inline src/prasad/Findiftwostrareindentical.java:12]: you can put condition  != condition and flag = false/true and then check flag value. Accordingly flag value print message,
Enhancement

Comment (PrasadCOct2020) [reply src/prasad/Findiftwostrareindentical.java]: Corrected the class please check

File: src/prasad/Stringandarray/Findcommonelefrarray.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+input:
+
+int array1 [] = [1, 2, 5, 5, 8, 9, 7, 10]
+int array2 [] = [1, 0, 6, 15, 6, 4, 7, 0]
+
+output: 1 7*/
+
+public class Findcommonelefrarray {
+	void scanArray(int[]array1 , int[]array2) {
```
Comment (harshvegada) [inline src/prasad/Stringandarray/Findcommonelefrarray.java:12]: Code works fine but time complexity very high
for(){
  while(){
  }
  for(){
   }
}

you can check this code

static void printCommonElement(int a[], int[] b) {
		for (int i = 0; i < a.length; i++) {
			for (int j = 0; j < b.length; j++) {
				if (a[i] == b[j]) {
					System.out.println(a[i]);
					break;
				}
			}
		}
	}

**Review by harshvegada — APPROVED** [review]
Update code & acknowledge me!

**Pull request merged**